sts_synindex_external.R

```diff
@@ -112,83 +118,77 @@ replace_equal_with_underscore <- function(directory_path) {
   }
 }
 
-invisible(lapply(list.dirs(PARQUET_FINAL_LOCATION), replace_equal_with_underscore))
-
-# Generate manifest of existing files
-sync_cmd <- glue::glue('aws s3 --profile service-catalog sync {base_s3_uri_archive} ./archive --exclude "*owner.txt*" --exclude "*archive*"')
-system(sync_cmd)
+invisible(lapply(list.dirs(AWS_ARCHIVE_DOWNLOAD_LOCATION), replace_equal_with_underscore))
```

Comment:
I'm assuming that the reason you have this function is because synapse doesn't support = in the folder name. I filed a ticket in platform.  As you add these workarounds: think about what could be added to synapse to avoid these.

https://sagebionetworks.jira.com/browse/PLFM-8091

Separately, why invisible? Could you save this to a variable. If this is the chunk of code that is breaking, it would be harder to debug with 'invisible'

Reply:
Yes the equal sign is incompatible with Synapse object naming protocol. Thanks for bringing that up; I will make note of bugs and features that I think could be updated in Synapse in the future.

Re: using invisible, this was to temporarily disable printing of the returned object. Your suggestion of storing the output to a variable is a more ideal solution though so I have implemented that change in commit [70b0bc9](https://github.com/Sage-Bionetworks/recover-parquet-external/pull/4/commits/70b0bc9c69708e8b5b4a4e550f36f2ba48100c60).